pipeline/patch_VAE.py

```diff
@@ -86,6 +88,69 @@ def build_trajectories(paths):
     return
 
 
+def generate_trajectory_relations(paths):
```

Comment:
I added this function to `generate_trajectories.py` for better consistency, and the step of generating relation matrix is now attached with the main pipeline.